README.md

```diff
@@ -74,88 +74,88 @@ You can also create a [manifest parser plugin][] to support custom manifest form
+ - MPD@timeShiftBufferDepth for seeking backward in Live streams
+ - Multi-period content (static and dynamic)
+ - Xlink elements (actuate=onLoad only, resolve-to-zero, fallback content)
+ - All forms of segment index info: SegmentBase@indexRange, SegmentTimeline, SegmentTemplate@duration, SegmentTemplate@index, SegmentList
+ - Multi-codec/multi-container manifests (we will negotiate support with the browser and choose the best ones)
+ - Encrypted content (including custom ContentProtection schemas, PSSH in the manifest)
+ - Key rotation
+ - Trick mode tracks
 
 DASH features **not** supported:
- - Xlink with actuate=onRequest
- - Manifests without any segment info: https://github.com/google/shaka-player/issues/1088
- - Changing codecs during a presentation (unsupported by MSE)
- - Multiple trick mode tracks for the same resolution at varying framerates or bitrates
- - Timescales so large that timestamps cannot be represented as integers in JavaScript (2^53): https://github.com/google/shaka-player/issues/1667
+ - Xlink with actuate=onRequest
+ - Manifests without any segment info: https://github.com/google/shaka-player/issues/1088
+ - Changing codecs during a presentation (unsupported by MSE)
+ - Multiple trick mode tracks for the same resolution at varying framerates or bitrates
+ - Timescales so large that timestamps cannot be represented as integers in JavaScript (2^53): https://github.com/google/shaka-player/issues/1667
 
 
 ## HLS features
 
 HLS features supported:
- - VOD, Live, and Event types
- - Encrypted content with Widevine
- - ISO-BMFF / MP4 / CMAF support
- - MPEG-2 TS support (transmuxing provided by [mux.js][] v5.1.3+, must be separately included)
- - WebVTT and TTML
- - CEA-608/708 captions
- - Encrypted content with FairPlay (Safari on macOS and iOS only)
+ - VOD, Live, and Event types
+ - Encrypted content with Widevine
+ - ISO-BMFF / MP4 / CMAF support
+ - MPEG-2 TS support (transmuxing provided by [mux.js][] v5.1.3+, must be separately included)
+ - WebVTT and TTML
+ - CEA-608/708 captions
+ - Encrypted content with FairPlay (Safari on macOS and iOS only)
 
 HLS features **not** supported:
- - Encrypted content with PlayReady: https://github.com/google/shaka-player/issues/1145
- - Key rotation: https://github.com/google/shaka-player/issues/917
- - I-frame-only playlists: https://github.com/google/shaka-player/issues/742
- - Raw AAC (without an MP4 container): https://github.com/google/shaka-player/issues/1083
+ - Encrypted content with PlayReady: https://github.com/google/shaka-player/issues/1145
+ - Key rotation: https://github.com/google/shaka-player/issues/917
+ - I-frame-only playlists: https://github.com/google/shaka-player/issues/742
+ - Raw AAC (without an MP4 container): https://github.com/google/shaka-player/issues/1083
 
 [mux.js]: https://github.com/videojs/mux.js/releases
 
 
 ## DRM support matrix
 
-|Browser   |Widevine  |PlayReady|FairPlay |ClearKey⁶ |
+|Browser   |Widevine  |PlayReady|FairPlay |ClearKey⁶ |
```

Comment:
I don't see what you've changed in the tables.  From GitHub, it appears that you may have added or removed a single space, but it's hard to tell.  Please avoid reformatting whitespace.  It makes it very hard to spot the meaningful, non-cosmetic changes.